docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -1,24 +1,24 @@
 ---
 title: Browsers used by Office Add-ins
 description: 'Specifies how the operating system and Office version determine what browser is used by Office Add-ins.'
-ms.date: 06/01/2020
+ms.date: 07/07/2020
 localization_priority: Normal
 ---
 
 # Browsers used by Office Add-ins
 
-Office add-ins are web applications that are displayed using iFrames when running in Office on the web and using embedded browser controls in Office for desktop and mobile clients. Add-ins also need a JavaScript engine to run the JavaScript. Both the embedded browser and the engine are supplied by a browser installed on the user's computer.
+Office add-ins are web applications that are displayed using iFrames when running in Office for the web and using embedded browser controls in Office for desktop and mobile clients. Add-ins also need a JavaScript engine to run the JavaScript. Both the embedded browser and the engine are supplied by a browser installed on the user's computer.
```

Comment:
See email thread on "for" vs "on" discussion.

Reply:
Reverted. Agree in context "on" is better than "for." Plus, there's the inconsistency with "Office on Mac."